import-export-cli/cmd/undeploy.go

```diff
@@ -29,9 +29,9 @@ const undeployCmdShortDesc = "Undeploy an API/API Product revision from a gatewa
 
 const undeployCmdLongDesc = `Undeploy an API/API Product revision available in the environment specified by flag (--environment, -e) from the gateway specified by flag (--gateway, -g)`
 
-const undeployCmdExamples = utils.ProjectName + ` ` + UndeployCmdLiteral + ` ` + UndeployAPICmdLiteral + ` -n TwitterAPI -v 1.0.0 -r admin --rev 1 -g Label1 Label2 -e dev
+const undeployCmdExamples = utils.ProjectName + ` ` + UndeployCmdLiteral + ` ` + UndeployAPICmdLiteral + ` -n TwitterAPI -v 1.0.0 -r admin --rev 1 -g Label1 -g Label2 -e dev
```

Comment:
Shall we add an example of a gateway label with two words which needs to wrap with quotes?

Reply:
Actually, as I have checked we cannot add Gateway environments with spaces from the Admin portal. For example, we cannot add something like "Production and Sandbox". So, that means there will not be a use case like that I think. @renuka-fernando and @uvindra any thoughts on this?

Reply:
I also think we do not explicitly have an example for this, since users generally know to assign a value to a flag or arg with spaces, they have to use quotes,  ex: `mkdir "hello world"`
But we can add gateway names with spaces using deployment.toml.